docs/src/main/sphinx/client/jdbc.md

```diff
@@ -246,3 +246,10 @@ may not be specified using both methods.
   - Sets the time zone for the session using the [time zone
     passed](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/ZoneId.html#of(java.lang.String)).
     Defaults to the timezone of the JVM running the JDBC driver.
+* - `legacyPreparedStatements`
+  - When set to `false`, prepared statements are executed calling a single
+    `EXECUTE IMMEDIATE` query instead of the standard `PREPARE <statement>`
+    followed by `EXECUTE <statement>`. Note that calling
```

Comment:
Remove this sentence from "Note" onwards since this is not user facing.

Reply:
Done